style.css

```diff
@@ -0,0 +1,179 @@
+:root {
+  --header-mobile-color: #3a4a42;
+  --header-desktop-color: #344563;
+  --headline-color-1: #172b4d;
+  --headline-color-2: #36b37e;
+  --background-color: #fff;
+}
+
+/* Reset the default browser styles */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+/* Global styles */
+body {
+  background: var(--background-color);
+}
+
+/* Navbar */
+nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: 48px;
+}
+
+/* Hide the nav bar on mobile */
+.navbar ul {
+  display: none;
+}
+
+/* Hamburger menu */
+.hamburger_toggle {
+  display: block;
+  cursor: pointer;
+  margin-right: 1rem;
+  margin-top: 0.75rem;
+  align-items: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  width: 2rem;
+  height: 1.75rem;
+  background: var(--background-color);
+  border: none;
+}
+
+.navbar li {
+  margin: 0 1rem;
+}
+
+.navbar li a {
+  color: var(--header-mobile-color);
+}
+
+/* Section headline */
+.headline {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+}
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓